cloud/azure/deploytf/.nitric/modules/cdn/main.tf

```diff
@@ -0,0 +1,144 @@
+locals {
+  endpoint_name               = "${var.stack_name}-cdn"
+  default_origin_group_name   = "${var.stack_name}-default-origin-group"
+  default_origin_name         = "${var.stack_name}-default-origin"
+
+  changed_path_md5_hashes     = join("", sort(values(var.uploaded_files)))
+}
+
+# Create the CDN profile for the website
+resource "azurerm_cdn_frontdoor_profile" "cdn_profile" {
+  name                = "${var.stack_name}-cdn-profile"
+  resource_group_name = var.resource_group_name
+  sku_name            = "Standard_AzureFrontDoor"
+}
+
+resource "azurerm_cdn_frontdoor_endpoint" "cdn_endpoint" {
+  name                     = local.endpoint_name
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.cdn_profile.id
+}
+
+resource "azurerm_cdn_frontdoor_origin_group" "default_origin_group" {
+  name                     = local.default_origin_group_name
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.cdn_profile.id
+
+  load_balancing {
+    additional_latency_in_milliseconds = 200
+    sample_size                        = 5
+    successful_samples_required        = 3
+  }
```

Comment:
I'd include the same documentation you included for pulumi for these values and possibly some azure documentation as well for future reference to save confusion for future readers.